packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -63,7 +58,12 @@ MeetingInfoUtil.generateOptions = (from) => {
     };
   }
   const options = {};
-  if (MeetingInfoUtil.isSipUri(destination)) {
+  // TODO: convert to sip URI?
```

Comment:
Is there a particular "default" format you find is most useful to pass around internally?